cypress/e2e/map/map.cy.js

```diff
@@ -1253,7 +1287,7 @@ describe('Map interactions', () => {
           cy.wait(`@${alias}`)
         })
 
-        cy.url().should('include', '?lat=14.410148015647792')
+        cy.url().should('include', '?lat=')
```

Comment:
Does cypress support matching like `.should('match', /\?lat=\d{2}(\.\d+)?/ig)` ?

Reply:
I tried it out and it does support your example. The problem here was the old code moved it to 14.4###, and the new code moved it to 15.###. I don't want to just accept any value as correct, but I shortened it to just accepting `lat=15` to ensure the map is moved to the right position